scipy/optimize/_bracket.py

```diff
@@ -10,43 +11,53 @@ def _bracket_root_iv(func, xl0, xr0, xmin, xmax, factor, args, maxiter):
     if not callable(func):
         raise ValueError('`func` must be callable.')
 
-    if not np.iterable(args):
+    if not isinstance(args, tuple):
```

Comment:
Lucky guess - looks like this is the only problem for `stats.tests.test_continuous`.

Reply:
```suggestion
    if not np.iterable(args):
```